.azure-pipelines/client.test.live.yml

```diff
@@ -0,0 +1,25 @@
+variables:
+  - group: KeyVault Test Secrets
+  - name: ARM_CLIENTID
+    value: $(java-keyvault-test-arm-client-id)
+  - name: ARM_CLIENTKEY
+    value: $(java-keyvault-test-arm-client-key)
+
+resources:
+- repo: self
+queue:
+  name: Hosted VS2017
+steps:
+- bash: |
```

Comment:
Is there a reason you aren't using the Maven@3 task to run the tests similar to https://github.com/Azure/azure-sdk-for-java/blob/master/.azure-pipelines/client.test.yml#L17?

In general we should try to do that to help be consistent and also that allows us to capture the test results into the test tab of the build which gives us a good view of the tests than ran and a history of them.

Reply:
The environment were not read during the build when tried to configure as Maven task.